index.js

```diff
@@ -7,6 +7,27 @@ function defaultCondition() {
   return true
 }
 
+function clampPercentage(val) {
+  return Math.max(0, Math.min(100, +(val || 0)))
+}
+
+function normalizePercentageRange(val) {
+  if (val && typeof val === 'object') {
+    return {
+      min: clampPercentage(val.min),
+      max: clampPercentage(val.max)
+    }
+  }
+  return clampPercentage(val)
+}
+
+function isPercentageInRange(val, range) {
+  if (range && typeof range === 'object') {
+    return val > range.min && val <= range.max
```

Comment:
Seems more intuitive but this means the range will be `1-100` not `0-99`.
This should be noted somewhere.

Reply:
Technically, the range is `Number.MIN_VALUE` (5e-324) to 100, but yeah, zero will never be in range. I'll note that in the code and in the README.